README.md

```diff
@@ -68,6 +75,7 @@ After writing this method, our new result should be:
 ```
 
 **think**: Where should we call `strip_punctuation`? Does it matter? Why?
+I called in within the loop where I am asking it to return the date and resulting string, because it has to be called on a string, not an array of strings, and this is where I was already destructuring the array of strings.
```

Comment:
This is good reasoning. I should have also asked "Should you call strip_punctuation inside or outside analyze_mood? Why?" Your reasoning is good but I was also looking for the fact that there's probably no reason we would ever want to analyze_mood on text with punctuation, so we should call it inside the method.